File: server/aoNewsletter2/Models/Db/_BlankModel.vb
﻿Imports System
Imports Contensive.Addons.Newsletter.Models.Db
Imports Contensive.BaseClasses
Imports Contensive.Models.Db

Namespace Models.Db
    Public Class BlankModel
        Inherits Global.Contensive.Models.Db.DbBaseModel

        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Newsletter Ad Banner Layouts", "NewsletterAdBannerLayouts", "default", False)
        '
        Public Property imageFilename As String
        '
    End Class
End Namespace

File: server/aoNewsletter2/Models/Db/_BlankModel.cs
using Contensive.Models.Db;

namespace Contensive.Addons.Newsletter.Models.Db {
    public class BlankModel : DbBaseModel {

        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Newsletter Ad Banner Layouts", "NewsletterAdBannerLayouts", "default", false);
        // 
        public string imageFilename { get; set; }
        // 
    }
}